recipes/mmtf-python/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "mmtf-python" %}
+{% set version = "1.0.4" %}
+{% set sha256 = "8aa92bc66b055f952c88f2dc33cd5dd52e39df376bd8ad88cca1cc344b2990da" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - msgpack-python
+
+  run:
+    - python
+    - msgpack-python
+
+test:
+  imports:
+    - mmtf
+    - mmtf.api
+    - mmtf.codecs
+    - mmtf.codecs.decoders
+    - mmtf.codecs.encoders
+    - mmtf.converters
+    - mmtf.tests
+    - mmtf.utils
+
+about:
+  home: http://mmtf.rcsb.org
+  license: Apache-2.0
+  license_family: Apache
+  # will be added in the next version
+  # license_file: LICENSE.txt
```

Comment:
I saw the PR in the source project, but it was pointed out to me recently that Apache 2.0 requires re-distribution of the license (see section 4a). When this came up earlier today in a recipe I was developing, I copied the license into the recipe and then added it as in:

https://github.com/conda-forge/staged-recipes/pull/1971/files